clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStorageEngineFactory.java

```diff
@@ -280,7 +280,7 @@ public synchronized Set<String> getPersistedStoreNames() {
     File databaseDir = new File(rocksDBPath);
     if (databaseDir.exists() && databaseDir.isDirectory()) {
       String[] storeDirs = databaseDir.list();
-      LOGGER.info("Found the following RocksDB databases: {}", Arrays.toString(storeDirs));
```

Comment:
may be this one by default could be helpful to find how many DB the server found ?

Reply:
I'm not sure about this.. How are we going to use this information tho? The reason why I wanted to remove this is when a single host have too many replica, this log line would be huge and is hard to parse.

Reply:
The good thing is it only send out once for startup. Let me know what you think. We can keep it if you find it can be helpful for debugging.